west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 9fc99928caf04c3ae78871f803ce8cee83541ae0
+      revision: 7d54586751cfe405d2cdcfd850f8100bc9844f63
```

Comment:
Thanks for the detailed git commit message! I'm afraid you may need to udpate this now that we merged https://github.com/thesofproject/sof/pull/7642 but that PR was older and had the approvals plus tests pass, so I merged that first.

Reply:
No problem, I think that it's better if we do zephyr update in smaller jumps.